LaunchAtLogin/LaunchAtLogin.swift

```diff
@@ -1,21 +1,156 @@
+import Combine
 import Foundation
 import ServiceManagement
 
-public struct LaunchAtLogin {
-	private static let id = "\(Bundle.main.bundleIdentifier!)-LaunchAtLoginHelper"
+public class LaunchAtLogin {
```

Comment:
My thinking was that this could be a `enum LaunchAtLogin` (just as a namespace for the various things) and just expose the separate observable object class too.